db/schema.rb

```diff
@@ -10,11 +10,36 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_10_25_100428) do
+ActiveRecord::Schema.define(version: 2019_10_29_100540) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "comments", force: :cascade do |t|
+    t.text "body"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "user_id"
+    t.integer "post_id"
+  end
+
+  create_table "friendships", force: :cascade do |t|
+    t.bigint "user_id"
+    t.bigint "friend_id"
+    t.boolean "confirmed"
```

Comment:
- [ ] **[OPTIONAL]** The default value for the `confirmed` column will be always `nil`. While this may or may not affect the performance, it will definitely affect your code readability. I suggest to add a new migration to change the default value to `false` so you can easily deal with friendship table in future.

You can use active record method [`change_column_default`](https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/change_column_default) to add a new migration to change its default value to false.